e2e/test_module.py

```diff
@@ -18,10 +19,10 @@ class TestConstants:
     OUTPUT = "output"
     QUERY = "query"
     TEST_DIR_ENDING = "_test"
+    TEST_DIR_ONLINE_PREFIX = "test_online"
+    ONLINE_TEST = "_test"
```

Comment:
Why are there two different consts for the same thing: "_test"

Reply:
`ONLINE_TEST` and `TEST_DIR_ENDING`

Reply:
Because it's a well-noticed typo. The first is not being used.